app/controllers/concerns/idv/unavailable_concern.rb

```diff
@@ -0,0 +1,26 @@
+module Idv
+  class UnavailableError < StandardError; end
+
+  module UnavailableConcern
```

Comment:
Yeah, the logic in `RegistrationsController` was there previously--I think the idea is that, if we know the user will fail proofing because proofing is not available, don't even let them _start_ to sign up. Just show an error.